docs/docs/launch-manual/index.md

```diff
@@ -22,6 +22,13 @@ This is really what Pulsar is all about, here you will get to grips with what
 lies under the skin of Pulsar so you can begin hacking on it to extend and
 customize its functionality.
 
+## [<i class="fa-solid fa-microscope"></i> Behind Pulsar](/docs/launch-manual/sections/behind-pulsar)
+
+Her you will get into the real nitty-gritty of Pulsar. Here you will get into
```

Comment:
```suggestion
Here you will get into the real nitty-gritty of Pulsar, where you get into
```

Spelling, and also a change in language, just since the double 'Here you will get into the' sounded repetitive.

Reply:
Honestly no idea how that line left my editor. That was awful, thanks for fixing it.